legal-api/src/legal_api/services/pdf_service.py

```diff
@@ -21,9 +24,20 @@
 from reportlab.pdfbase.ttfonts import TTFont
 from reportlab.pdfgen import canvas
 
+from legal_api.reports.registrar_meta import RegistrarInfo
 from legal_api.utils.legislation_datetime import LegislationDatetime
 
 
+@dataclass
+class RegistrarStampData:
```

Comment:
Created so we can pass more than 4 arguments at once without making the function have way too many parameters